src/components/pager/pager.spec.js

```diff
@@ -31,22 +32,26 @@ function render(props, renderType = mount) {
 }
 
 describe('Pager', () => {
+  it('renders the Pager without pageSizeSelection', () => {
+    const wrapper = render({
+      totalRecords: 100,
+      pageSize: 10,
+      showPageSizeSelection: false,
+      onPagination: () => true,
+      theme: mintTheme
```

Comment:
theme should be passed with `ThemeProvider`

Reply:
it is, render method consumes it
```
function render(props, renderType = mount) {
  return renderType(
    <ThemeProvider theme={ props.theme }>
      <Pager { ...props } />
    </ThemeProvider>
  );
}```